lib/api/src/js/native.rs

```diff
@@ -64,11 +66,34 @@ macro_rules! impl_native_traits {
             pub fn call(&self, mut store: &mut impl AsStoreMut, $( $x: $x, )* ) -> Result<Rets, RuntimeError> where
             $( $x: FromToNativeWasmType + crate::js::NativeWasmTypeInto, )*
             {
-                let params_list: Vec<JsValue> = vec![ $( JsValue::from_f64($x.into_raw(&mut store))),* ];
```

Comment:
Can we add a new comment linking to the asyncify refactor afterwards:

```
// TODO: issue #XYZ 
```

Reply:
done